test/test_dataset.jl

```diff
@@ -63,6 +73,15 @@ import ArchGDAL as AG
             end
         end
 
+        for driver in supported_vector_drivers
```

Comment:
Here we basically read data, write it to disk, read it back in and compare layer and feature count with the original.